examples/1_Spectra_handling/plot_exomol_spectrum.py

```diff
@@ -13,7 +13,7 @@
 s = calc_spectrum(
     1080,
     1320,  # cm-1
-    molecule="SiO",
+    species="SiO",
```

Comment:
is the `molecule=` syntax still valid ? 

If not, it should be (even if not advertised for in the docs; but it should be read from the **kwargs and converted into "species", obviously raising an error if both "molecule=" and "species=" are given)

Reply:
I agree. We discussed it and I'd like the species argument to be still valid